### PR TITLE
Add two screenshots to Gyroscope Instrument

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [ ] Feature implementation
- [x] Doc updates

* **Related Issue**

* **What changes have you introduced?**
Added a screenshot of the gyroscope showing on the menu.
Added a screenshot of the gyroscope with a different reading.

* **Does this PR introduce a breaking change?**
No

* **Preview / Steps to verify your work**:
https://github.com/kumuditha-udayanga/pslab-documentation/blob/gyro/tutorials/gyroscope.md

### CODE REVIEW

**Comment by kumuditha-udayanga** [comment]
@cweitat please review 🙂

File: tutorials/gyroscope.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ well as how to use it.
 The layout consists of three different graph plots for X, Y, and Z axes. There is also the option to configure the Gyrometer
 using the menu in the top right corner. 
 
+<table>
+    <tr>
+        <td><img src="/_static/instrument_gyro_home.jpg"></td>
```
Comment (cweitat) [inline tutorials/gyroscope.md]: broken links

**Comment by kumuditha-udayanga** [comment]
Sure

File: tutorials/gyroscope.md (hunk 1 above)
Comment (NishithP2004) [reply tutorials/gyroscope.md]: Screenshot_2020-01-14-16-15-16-535_com android chrome
Sir could you please recheck as the images are appearing the fork

Comment (NishithP2004) [reply tutorials/gyroscope.md]: You may check his gyroscope.md file , the images are appearing

Comment (kumuditha-udayanga) [reply tutorials/gyroscope.md]: Preview link: https://github.com/kumuditha-udayanga/pslab-documentation/blob/gyro/tutorials/gyroscope.md

Comment (kumuditha-udayanga) [reply tutorials/gyroscope.md]: @cweitat it seems no any broken links. All images have appeared in the gyroscope.md file

**Comment by kumuditha-udayanga** [comment]
@NishithP2004 Thank You!

**Comment by kumuditha-udayanga** [comment]
@fcartegnie Please review :)

**Comment by kumuditha-udayanga** [comment]
@cweitat Please review :)

**Comment by kumuditha-udayanga** [comment]
@cweitat please review :)

File: tutorials/gyroscope.md (hunk 1 above)
Comment (cweitat) [reply tutorials/gyroscope.md]: can we try to have 4 images in 1 row?

Comment (kumuditha-udayanga) [reply tutorials/gyroscope.md]: Yes I tried then they are too small

Comment (kumuditha-udayanga) [reply tutorials/gyroscope.md]: Especially in the mobile version

**Comment by kumuditha-udayanga** [comment]
@fcartegnie please review 😇

**Comment by kumuditha-udayanga** [comment]
@mariobehling please review :)

**Comment by kumuditha-udayanga** [comment]
> Please move files to images folder.

Okay sure :)

**Comment by kumuditha-udayanga** [comment]
@mariobehling Changes have been done 😇, please review

**Comment by kumuditha-udayanga** [comment]
Thank you :)

**Pull request merged**